play/src/front/Components/Input/InputLabel.svelte

```diff
@@ -1,21 +1,52 @@
-<script>
-    export let id = "";
-    export let label = "";
-    export let value = "";
+<script lang="ts">
+    export let id: string;
+    export let label: string;
     export let placeholder = "";
     export let onChange = () => {};
     export let disabled = false;
+    export let options: { value: string | undefined; label: string }[] = [];
+    export let type: "text" | "select" | "url" = "text";
+    export let value: string | null | undefined;
+    export let onClick = () => {};
```

Comment:
Let's also add the "helpText" prop that displays the small text at the bottom.
And even better: a "info" slot to display the HTML to be displayed when the mouse is hovered the "i" button.

Reply:
And a "state" prop with "success"|"error"|"" type

Reply:
And a "before", "after" slot :) for the icons or buttons at the beginning or end of the field.

Reply:
Add a prop for the different sizes